new/PRS14050MentasnotAbbuhay.xml

```diff
@@ -0,0 +1,59 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="PRS14050MentasnotAbbuhay" xml:lang="en" type="pers">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title>Məntasnot ʾAbbuhay</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="AY"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
+            </abstract>
+            <t:langUsage xmlns:t="http://www.tei-c.org/ns/1.0">
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </t:langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2023-07-25">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <listPerson>
+                <person sex="1">
+                    <persName xml:lang="gez" xml:id="n1">ምንተስኖት፡ አቡሀይ፡</persName>
+                    <persName xml:lang="gez" type="normalized" corresp="#n1">Məntasnot ʾAbbuhay</persName>
+                    <faith type="EOTC"/>
+                    <occupation type="academic">He became a textile engineer from 1983/84 to 1998/99.</occupation>
```

Comment:
1) You say "he was an engineer from .... to", or "he worked as engineer", but not "He became". 
You should provide and encode the original dates after the Ethiopian calendar (EC). There is a possibility for that. To read statements like "He did smth from 1983/84 to 1998/99" is somewhat difficult. Try to apply `https://betamasaheft.eu/Guidelines/?q=Dates&id=dates` . For instance, it would be: `<occupation type="academic">He was a textile engineer from <date calendar="ethiopian" notBefore=”1983” notAfter=”1984” when-custom=”1976”/> to <date calendar="ethiopian" notBefore=”1998” notAfter=”1999” when-custom=”1991”/>`, I assumed you had the dates 1976 and 1991 EC in mind.